README.md

```diff
@@ -70,6 +70,7 @@ Tile38 can be compiled and used on Linux, OSX, Windows, FreeBSD, and probably ot
 
 To build everything simply:
 ```
+$ git tag <major>.<minor>.<fix>
```

Comment:
Is this line a mistake?

Reply:
Nope, the `make` will explode if there isn't a git tag at the head of the branch. Figured that was intentional. Since it took a bit to figure out the error message i figured i'd add it to the docs.

Reply:
I couldn't reproduce the issue on my side.
I try to keep the master branch green, and a simple `make` should always generate a working build. 

Running Ubuntu (amd64/linux) with Go 1.20.5:

```
$ git clone https://github.com/tidwall/tile38
$ cd tile38
$ make
$ ./tile38-server
   _____ _ _     ___ ___
  |_   _|_| |___|_  | . |  Tile38 1.31.0 (0144ca68) 64 bit (amd64/linux)
    | | | | | -_|_  | . |  Port: 9851, PID: 352653
    |_| |_|_|___|___|___|  tile38.com
```

There was no error.

Reply:
Ok, I pulled it.